src/effects/EffectUI.cpp

```diff
@@ -757,10 +757,7 @@ EffectUIHost::EffectUIHost(wxWindow *parent,
 
 EffectUIHost::~EffectUIHost()
 {
-   CleanupRealtime();
-   if (mNeedsResume)
-      Resume();
-   mClient.CloseUI();
+   wxASSERT(mClosed);
```

Comment:
`mClosed` is a new member field, which seems to be used only in debug builds, since `wxASSERT` macro does nothing in release builds. Couldn't the definition of the variable be made conditional too?

Reply:
As you wish